lang/de.json

```diff
@@ -1,1146 +1,1168 @@
+			"AdrenalineDetails": "Adrenalin ist der Wille weiter zu machen und zu kämpfen in fatalen Situationen.",
+			"displayRollDetails": "Zeige Würfelwurf Details",
+			"displayRollDetailsHint": "Zeigt Details im Chat bei einem Würfelwurf, z.B. 1d10+2[stat]+4[skill]-3[parry]",
+			"Custom": "Custom",
+			"modifiers": "Modifiers",
+			"displayReputation": "Zeige Spielern ihre Reputation an",
+			"displayReputationHint": "Erlaubt es Spielern ihre Reputation anzuzeigen und zu verändern.",
+			"useVerbalCombatRule": "Verwende die optionale Regel für verbalen Schlagabtausch",
+			"useVerbalCombatRuleHint": "Erlaubt es Spielern alle Aktionen für verbalen Schlagabtausch zu sehen.",
+			"clickableImageItemTypes": "Zeige 'Zoombar'-Checkbox bei Items",
+			"clickableImageItemTypesHint": "Eine kommaseparierte Liste von Item Typen, bei denen eine Checkbox angezeigt werden soll. Wenn die Checkbox angekreuzt ist, können die Spieler im Inventar auf das Item Bild klicken um es zu vergrößern. Mögliche Item Typen sind 'valuable', 'weapon', 'armor', 'alchemical', 'diagrams', 'component'.",
+			"clickableImageCheckboxForGMOnly": "Zeige 'Zoombar'-Checkbox nur dem GM",
+			"specialFont": "Benutze die besondere Schriftart von 'The Witcher'",
+			"globalModifierLookupCompendium": "Global Modifier Nachschlage Compendium",
+			"globalModifierLookupCompendiumHint": "Wenn ein Globaler Modifikator aktiviert werden soll, dieser aber nicht auf dem Charakterbogen vorhanden ist, wird in diesem Kompendium danach gesucht"
+		},
+		"background.other": "Andere",
+		"background.globalModifier": "Globale Modifikatoren",
+		"Resources.modifiers": "Ressourcenmodifikatoren",
+		"Notes": "Notizen",
+		"Skills": {
+			"SkillGroups": {
+				"allSkills": "Alle Fertigkeiten",
+				"meleeSkills": "Nahkampffertigkeiten",
+				"rangedSkills": "Fernkampffertigkeiten",
+				"magicSkills": "Magiefertigkeiten",
+				"verbalCombatSkills": "Verbaler Schlagabtausch Fertigkeiten",
+				"empatheticVerbalCombatSkills": "Einfühlsame Verbaler Schlagabtausch Fertigkeiten"
+			}
+		},
+		"SkIntAwareness": "Aufmerksamkeit",
+		"SkIntBusiness": "Geschäftsinn",
+		"SkIntDeduction": "Schlussfolgerunden",
+		"SkIntEducation": "Bildung",
+		"SkIntCommon": "Gemeinsprache(2)",
+		"SkIntCommonLable": "Gemeinsprache",
+		"SkIntElder": "Alte Sprache (2)",
+		"SkIntElderLable": "Alte Sprache",
+		"SkIntDwarven": "Zwergisch (2)",
+		"SkIntDwarvenLable": "Zwergisch",
+		"SkIntMonster": "Ungeheuerkunde (2)",
+		"SkIntMonsterLable": "Ungeheuerkunde",
+		"SkIntSocialEt": "Umgangsformen",
+		"SkIntStreet": "Straßenwissen",
+		"SkIntTactics": "Taktik (2)",
+		"SkIntTacticsLable": "Taktik",
+		"SkIntTeaching": "Unterrichten",
+		"SkIntWilderness": "Wildnisleben",
+		"SkRefBrawling": "Raufen",
+		"SkRefDodge": "Ausweichen/Ausbrechen",
+		"SkRefMelee": "Nahkampf",
+		"SkRefRiding": "Reiten",
+		"SkRefSailing": "Segeln",
+		"SkRefSmall": "Kleine Klingenwaffen",
+		"SkRefStaff": "Stab/Speer",
+		"SkRefSwordmanship": "Schwertkampf",
```

Comment:
You cannot change this without touching every piece of code where this is used

Reply:
But shouldn't this only Touch the Displayed Translation in the skull Tab? The refetence is not foundry there. It's Displayed as "SkRefSwordmanship".

I will check and test local and will add the missing en file.

Thx for the Feedback

Reply:
Ah ok, I recognized that it's not properly translated in other passages. Quick solution would be including both. Opinion?

Like:

"SkRefSwordmanship": "Schwertkampf",
"SkRefSwordsmanship": "Schwertkampf",

(could do the same for the english one)

Reply:
I did a full text search and ``SkRefSwordmanship`` never appears. Adding it will have no effect. So I just would stick with the one with the extra ``s``. The labels need a rework anyway and it can be fixed then